core/src/main/scala/org/apache/spark/util/ThreadUtils.scala

```diff
@@ -133,6 +133,14 @@ private[spark] object ThreadUtils {
     Executors.newCachedThreadPool(threadFactory).asInstanceOf[ThreadPoolExecutor]
   }
 
+  /**
+   * Wrapper over newCachedThreadPool. Thread names are formatted as prefix-ID, where ID is a
```

Comment:
This doesn't really have a prefix in the name. Can we inline this method? It's only called once.

Reply:
Done